src/screens/NewsScreen/Search.js

```diff
@@ -45,7 +45,7 @@ Search.propTypes = {
 const styles = StyleSheet.create({
   container: {
     flexDirection: 'row',
-    backgroundColor: 'white',
+    backgroundColor: COLORS.white,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from a hardcoded color value to a reference from the `COLORS` object is a good practice. It improves maintainability and consistency across the application.
```diff
-    backgroundColor: 'white',
+    backgroundColor: COLORS.white,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->